test/use-swr-cache.test.tsx

```diff
@@ -15,49 +15,32 @@ describe('useSWR - cache', () => {
     }
 
     // render using custom cache
-    const { queryByText } = render(
+    render(
       <React.Suspense fallback={null}>
         <Page />
       </React.Suspense>
     )
 
     // content should come from custom cache
-    expect(queryByText('custom cache message')).toMatchInlineSnapshot(`
-      <div>
-        custom cache message
-      </div>
-    `)
+    expect(await screen.findByText('custom cache message')).toBeInTheDocument()
```

Comment:
if ```await screen.findByText('custom cache message')``` does not throw,  it means that the text content you are looking for is in the document. You don't need do another assertion here.

Reply:
@promer94 
Thank you for your feedback! That's right.
But I prefer to have an assert function explicitly to indicate this line is an assertion.
What about writing like this?

```js
await expect(screen.findByText('custom cache messagea')).resolves.toBeTruthy()
```

This is the way `dom-testing-lijbrary` is using in the internal tests.
https://github.com/testing-library/dom-testing-library/blob/master/src/__tests__/queries.find.js#L41

(I don't have a strong opinion for this, so it's ok to change the way not using an assert function )

Reply:
```js
await expect(screen.findByText('custom cache messagea')).resolves.toBeTruthy()
```
In this way, we are actually testing the findByText API rather than our component.

Reply:
Thank you!
I've fixed it at https://github.com/vercel/swr/pull/932/commits/d40345256c744a897d93c7b44222c29e19bbad89